### PR TITLE
Relax pip filtering to find additional HCLK_IOI3 pip bits.

### PR DESCRIPTION
**Pull request by litghost**
Solves 4 of the remaining 10 HCLK_IOI3 bits.

### CODE REVIEW

**Comment by kgugala** [comment]
I did fast check and the previous version of the fuzzer was ignoring those PIP that are used by Litex design:

```
HCLK_IOI3_X113Y26/HCLK_IOI3.HCLK_IOI_CK_BUFHCLK7->>HCLK_IOI_CK_IGCLK7
HCLK_IOI3_X113Y26/HCLK_IOI3.HCLK_IOI_CK_BUFHCLK11->>HCLK_IOI_CK_IGCLK11
HCLK_IOI3_X113Y26/HCLK_IOI3.HCLK_IOI_CK_BUFHCLK10->>HCLK_IOI_CK_IGCLK10
HCLK_IOI3_X113Y26/HCLK_IOI3.HCLK_IOI_CK_BUFHCLK9->>HCLK_IOI_CK_IGCLK9
```

**Comment by kgugala** [comment]
so the changes introduced in the PR should fix those

File: fuzzers/000-init-db/Makefile [hunk 1]
```diff
@@ -39,10 +39,11 @@ DB_SIMPLE=\
...
     segbits_cmt_top_r_upper_t \
-	segbits_lioi3 \
-	segbits_rioi3 \
-	segbits_liob33 \
-	segbits_riob33
+    segbits_lioi3 \
+    segbits_rioi3 \
+    segbits_liob33 \
+    segbits_riob33
+    segbits_hclk_ioi3 \
```
Comment (kgugala) [inline fuzzers/000-init-db/Makefile]: backslashes seem to be messed here

Comment (litghost) [reply fuzzers/000-init-db/Makefile]: Opps, merge conflict error, fixed.

**Pull request merged**